README.md

```diff
@@ -4,23 +4,27 @@ See https://w3c.github.io/rdf-star/
 
 ## Preliminary implementations ##
 
-| [SA] | [PG] | Implementation | Notes - Documentation
-|------|------|----------------|----------------------
-|      |  +   | AllegroGraph   | in the works - https://lists.w3.org/Archives/Public/public-rdf-star/2020Aug/0021.html
-|      |  x   | AnzoGraph      | https://docs.cambridgesemantics.com/anzograph/v2.2/userdoc/lpgs.htm?Highlight=rdf
-|      |  x   | BlazeGraph     | https://github.com/blazegraph/database/wiki/Reification_Done_Right
-|  x   |      | GraphDB        | http://graphdb.ontotext.com/documentation/9.2/free/devhub/rdf-sparql-star.html
-|  x   |      | Jena           | https://jena.apache.org/documentation/rdfstar/
-|  x   |  X   | EYE            | SA is the default. PG can be emulated by adding rules - https://github.com/josd/eye/
-|  x   |      | rdf4j          | https://rdf4j.org/documentation/programming/rdfstar/
-|  x   |  +   | rdfjs/N3.js    | PG may become configurable soon - https://github.com/rdfjs/data-model-spec/pull/165
-|  x   |  x   | RubyRDF        | https://github.com/ruby-rdf/rdf#rdf-rdfstar
-|      |  x   | Stardog        | https://www.stardog.com/docs/#_edge_properties
-|      |  x   | Corese         | http://ns.inria.fr/sparql-extension/rdfstar.html
+| Implementation | [SA] | [PG] | Notes | Documentation |
+|:---------------|:----:|:----:|:------|:-------------:
+| AllegroGraph   |      |  +   | in the works | [docs](https://lists.w3.org/Archives/Public/public-rdf-star/2020Aug/0021.html)
```

Comment:
I prefer to leave the links showing.

I find "docs" on every row less useful and especially as the AllegroGraph one isn't their final documentation.

Reply:
The links remain available, but I see your point about AllegroGraph.  I'll un-pretty the docs links.